gator/invoke.py

```diff
@@ -285,6 +285,80 @@ def invoke_all_fragment_checks(
     return met_or_exceeded_count
 
 
+def invoke_all_regex_checks(
+    fragment,
+    expected_count,
+    filecheck=NOTHING,
+    directory=NOTHING,
+    contents=NOTHING,
+    exact=False,
+):
+    """Perform the check for a regex existence in file or contents and return the results"""
+    met_or_exceeded_count = 0
+    met_or_exceeded_count, actual_count = fragments.specified_fragment_greater_than_count(
+        fragment,
+        fragments.count_specified_regex,
+        expected_count,
+        filecheck,
+        directory,
+        contents,
+        exact,
+    )
+    # create a message for a file in directory
+    if filecheck is not NOTHING and directory is not NOTHING:
+        if exact is not True:
+            message = (
+                "The "
+                + filecheck
+                + " in "
+                + directory
+                + " has at least "
+                + str(expected_count)
+                + " of the '"
+                + fragment
```

Comment:
The variable should be called regex and this message should probably mention "matching", you can't simply reuse `fragment`'s message.

Reply:
@everitt-andrew the variable should also be renamed so that future developers are not confused.